apps/coscrad-frontend/src/components/resources/Terms/TermIndex.container.tsx

```diff
@@ -0,0 +1,17 @@
+import { RootState } from '../../../store';
+import { fetchTerms } from '../../../store/slices/resources';
+import { useLoadable } from '../../../utils/custom-hooks/useLoadable';
+import { displayLoadableWithErrorsAndLoading } from '../../higher-order-components';
+import { TermIndexPresenter } from './TermIndex.presenter';
+
+export const TermIndexContainer = (): JSX.Element => {
+    const [loadableTerms] = useLoadable({
+        selector: (state: RootState) => state.terms,
+        fetchThunk: fetchTerms,
+    });
+
+    // Double check typesafety in places like this
```

Comment:
Do this.

Reply:
done. We may want to be careful with the generics in displayLoadableWithErrorsAndLoading going forward.